ironfish/src/rpc/routes/wallet/importAccount.ts

```diff
@@ -52,6 +56,11 @@ router.register<typeof ImportAccountRequestSchema, ImportResponse>(
   ImportAccountRequestSchema,
   async (request, node): Promise<void> => {
     let createdAt = null
```

Comment:
We will need to update this once we support string imports. We will need to check:
```
    const accountImport = AccountEncoder.decode(account)
    const name = accountImport.name || request.data.name
    if (!name) {
      throw new ImportError('Account name is required')
    }
```